migrate-demo-test.js

```diff
@@ -1,10 +1,11 @@
 const utils = require('./utils.js')
 async function assignGlobalVariables (web3, spinner, opts, logTx, base) {
+  this.arcVersion = require('./package.json').dependencies['@daostack/arc']
   this.web3 = web3
   this.spinner = spinner
   this.opts = opts
   this.logTx = logTx
-  this.base = base
+  this.base = base[this.arcVersion]
```

Comment:
so the demo test dao is always build on top of latest arc?

Reply:
Yes.

Reply:
it will be nice to have a demo dao build on privious versions (which are supported by the migration ) so it can be used by other layer for testing.

Reply:
ok I added that.